dashboard/test/models/script_test.rb

```diff
@@ -2875,6 +2875,51 @@ class MigratedScriptCopyTests < ActiveSupport::TestCase
       assert_equal '2022', cloned_unit.version_year
     end
 
+    test 'can update markdown on clone' do
+      resource = create :resource, course_version: @standalone_unit.get_course_version, name: 'resource', url: 'code.org'
+      vocab = create :vocabulary, course_version: @standalone_unit.get_course_version, word: 'word', definition: 'definition'
+      new_course_version = create :course_version, :with_course_offering
+      test_locale = :en
+      I18n.locale = test_locale
+      mock_i18n = {
+        'data' => {
+          'script' => {
+            'name' => {
+              @standalone_unit.name => {
+                'description_short' => "Description short: Resource: [r #{Services::GloballyUniqueIdentifiers.build_resource_key(resource)}]. Vocab: [v #{Services::GloballyUniqueIdentifiers.build_vocab_key(vocab)}].",
+                'description_audience' => "Description audience: Resource: [r #{Services::GloballyUniqueIdentifiers.build_resource_key(resource)}]. Vocab: [v #{Services::GloballyUniqueIdentifiers.build_vocab_key(vocab)}].",
+                'description' => "Description: Resource: [r #{Services::GloballyUniqueIdentifiers.build_resource_key(resource)}]. Vocab: [v #{Services::GloballyUniqueIdentifiers.build_vocab_key(vocab)}].",
+                'student_description' => "Student description: Resource: [r #{Services::GloballyUniqueIdentifiers.build_resource_key(resource)}]. Vocab: [v #{Services::GloballyUniqueIdentifiers.build_vocab_key(vocab)}].",
+              }
+            }
+          }
+        }
+      }
+      I18n.backend.store_translations test_locale, mock_i18n
+      copied_resource = resource.copy_to_course_version(new_course_version)
+      copied_vocab = vocab.copy_to_course_version(new_course_version)
+      expected_i18n = {
+        'en' => {
+          'data' => {
+            'script' => {
+              'name' => {
+                'new_name' => {
+                  'title' => '',
+                  'description_short' => "Description short: Resource: [r #{Services::GloballyUniqueIdentifiers.build_resource_key(copied_resource)}]. Vocab: [v #{Services::GloballyUniqueIdentifiers.build_vocab_key(copied_vocab)}].",
+                  'description_audience' => "Description audience: Resource: [r #{Services::GloballyUniqueIdentifiers.build_resource_key(copied_resource)}]. Vocab: [v #{Services::GloballyUniqueIdentifiers.build_vocab_key(copied_vocab)}].",
+                  'description' => "Description: Resource: [r #{Services::GloballyUniqueIdentifiers.build_resource_key(copied_resource)}]. Vocab: [v #{Services::GloballyUniqueIdentifiers.build_vocab_key(copied_vocab)}].",
+                  'student_description' => "Student description: Resource: [r #{Services::GloballyUniqueIdentifiers.build_resource_key(copied_resource)}]. Vocab: [v #{Services::GloballyUniqueIdentifiers.build_vocab_key(copied_vocab)}].",
+                  'lessons' => {}
```

Comment:
I'm not sure if there is a real risk here, but from a readability standpoint it's a bit hard to convince myself that the vocab key generated by `build_vocab_key(copied_vocab)` is in fact the right expected value. would you consider  using the plaintext string for the resource/vocab keys here, or maybe asserting that the output of `build_vocab_key(copied_vocab)` is what you expect?

Reply:
That makes sense. The key of the copied resource/vocab isn't necessarily deterministic, so it's a bit tough to check in plain text. I'll take a stab at making these more readable

Reply:
I'm not sure what's best here, but perhaps just checking that the version year has changed would be enough? I think the pitfall which comes to mind is that somehow `build_vocab_key(copied_vocab)` is the same as `build_vocab_key(vocab)` and therefore the markdown also does not change.

Reply:
the updates look great, thank you for tackling that!